Assets/Mirror/Runtime/ClientScene.cs

```diff
@@ -378,13 +379,16 @@ internal static void ApplySpawnPayload(NetworkIdentity identity, SpawnMessage ms
 
             NetworkIdentity.spawned[msg.netId] = identity;
 
-            // objects spawned as part of initial state are started on a second pass
-            if (isSpawnFinished)
-            {
-                identity.NotifyAuthority();
-                identity.OnStartClient();
-                CheckForLocalPlayer(identity);
-            }
+            // call OnStartAuthority, OnStartClient, OnStartLocalPlayer
```

Comment:
I like this,  and the removal of OnObjectSpawnStarted/OnObjectSpawnFinished.

I think we should have this in it's own PR though.